app/views/templates/invoices/v2.slim

```diff
@@ -567,6 +567,12 @@ html
                               - else
                                 = I18n.t('invoice.total_unit', units: fee.units)
                         td.body-1 width="30%" = fee.amount.format(format: I18n.t('money.format'), decimal_mark: I18n.t('money.decimal_mark'), thousands_separator: I18n.t('money.thousands_separator'))
+                      - if fee.true_up_fee.present?
+                      tr
+                        td width="70%"
+                          .body-1 = I18n.t('invoice.true_up_metric', metric: fee.billable_metric.name)
+                          .body-3 = I18n.t('invoice.true_up_details', min_amount: fee.charge.min_amount.format(format: I18n.t('money.format'), decimal_mark: I18n.t('money.decimal_mark'), thousands_separator: I18n.t('money.thousands_separator')))
+                        td.body-1 width="30%" = fee.true_up_fee.amount.format(format: I18n.t('money.format'), decimal_mark: I18n.t('money.decimal_mark'), thousands_separator: I18n.t('money.thousands_separator'))
```

Comment:
We will have to add a `localized_format` helper somewhere, because all these `format(format: I18n.t('money.format'), decimal_mark: I18n.t('money.decimal_mark'), thousands_separator: I18n.t('money.thousands_separator'))` makes me a bit crazy :exploding_head: